src/main.cpp

```diff
@@ -627,6 +627,15 @@ int main(int argc, char** argv)
     /* Start the event listener */
     drakvuf->loop(timeout);
 
+    PRINT_DEBUG("Beginning stop plugins\n");
+
+    bool plugin_stop_list[] = {[0 ... __DRAKVUF_PLUGIN_LIST_MAX-1] = 0};
+    plugin_stop_list[PLUGIN_BSODMON] = 1;
+    if (drakvuf->stop_plugins(plugin_stop_list) < 0)
+        return drakvuf_exit_code_t::FAIL;
+
+    PRINT_DEBUG("Finished stop plugins\n");
+
```

Comment:
Very nice, but what's the point? 2 lines below `drakvuf->terminate` already does call `delete` on all plugins...

Reply:
They should work after restart just the same way drakvuf itself should work after stopping/starting again. I can imagine this being part of a setup where you want to start/stop a plugin based on some event. For example one plugin being the coordinator watching for base events and starting/stopping some other plugins based on what it sees. You could also extend the signal processing interface to receive a kick from some other process even. So generally speaking I think this is a nice to have.

Reply:
Hmmm, it was a bad idea to put both things into 1 comment.
1) I agree, that it's a nice feature to be able to start & stop plugins (even if unused yet).
2) I don't see the point of the code above, since it would work exactly the same without it.